internal/worker_versioning_info.go

```diff
@@ -237,6 +237,15 @@ func taskQueueVersionInfoFromResponse(response *taskqueuepb.TaskQueueVersionInfo
 	}
 }
 
+func detectEnhancedNotSupported(response *workflowservice.DescribeTaskQueueResponse) error {
```

Comment:
Not sure this needs to be a separate top-level function in a separate file, but at least qualify it with "taskQueue" or something in the name since it shares the function space with the rest of the package.

Reply:
Makes sense, renaming...